setup.py

```diff
@@ -13,7 +13,12 @@ def parse_requirements():
                 # Handle custom index URLs
                 _, url = line.split()
                 _dependency_links.append(url)
-            elif "flash-attn" not in line and line and line[0] != "#":
+            elif (
+                "flash-attn" not in line
```

Comment:
Is there a reason to ignore deepspeed?

Reply:
deepspeed doesn't consistently install on all environments cleanly, esp WSL

Reply:
Ah I see